wsd/AdminModel.cpp

```diff
@@ -1044,9 +1057,9 @@ class ActiveExpiredStats final
 
     void Print(std::ostream& oss, const char* prefix, const char* name, const char* unit) const
     {
-        _all.Print(print(oss, prefix, "all", name), std::string_view(), unit);
-        _active.Print(print(oss, prefix, "active", name), std::string_view(), unit);
-        _expired.Print(print(oss, prefix, "expired", name), std::string_view(), unit);
+        _all.Print(oss, concat(prefix, name, "all"), unit);
```

Comment:
Is it intentional to swap `name` and `"all"` ?

Reply:
Well spotted this was a mistake, leftover from my investigation.